versioned/tests/src/main/java/org/projectnessie/versioned/tests/CommitBuilder.java

```diff
@@ -167,9 +168,12 @@ public Hash toBranch(String branchName)
    */
   public Hash toBranch(BranchName branchName)
       throws ReferenceNotFoundException, ReferenceConflictException {
-    Optional<Hash> reference = fromLatest ? Optional.of(store.toHash(branchName)) : referenceHash;
+    Optional<Hash> reference =
+        fromLatest
+            ? Optional.of(store.getNamedRef(branchName, GetNamedRefsParams.DEFAULT).getHash())
```

Comment:
`store.getNamedRef(branchName, GetNamedRefsParams.DEFAULT).getHash()` is being used twice, maybe extract into a variable?

Reply:
updated